Source/Details/ASTraitCollection.h

```diff
@@ -175,7 +181,7 @@ AS_SUBCLASSING_RESTRICTED
 @property (nonatomic, assign, readonly) UIUserInterfaceStyle userInterfaceStyle;
 #endif
 
-@property (nonatomic, assign, readonly) UIContentSizeCategory preferredContentSizeCategory;
+@property (nonatomic, assign, readonly) UIContentSizeCategory _Nonnull preferredContentSizeCategory;
```

Comment:
I don't think we need `_Nonnull` annotations since we already use `NS_ASSUME_NONNULL_BEGIN` at the beginning of this file ([here](https://github.com/TextureGroup/Texture/pull/757/files#diff-d9819ce020f3052ee72d0d87446ba684R27)). It's always good to be defensive and guard against unexpected values in the implementation though.